ui/src/components/admin/Workers.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+    <div>
+        <nav>
+            <collapse>
+                <el-form-item>
+                    <refresh-button class="float-right" @refresh="loadData" />
+                </el-form-item>
+            </collapse>
+        </nav>
+        <el-table
+            :data="workers"
+            ref="table"
+            :default-sort="{prop: 'hostname', order: 'ascending'}"
+            stripe
+            table-layout="auto"
+            fixed
+        >
+            <el-table-column prop="workerUuid" sortable :sort-orders="['ascending', 'descending']" :label="$t('id')" />
+            <el-table-column prop="hostname" sortable :sort-orders="['ascending', 'descending']" :label="$t('hostname')" />
+            <el-table-column prop="workerGroup" sortable :sort-orders="['ascending', 'descending']" :label="$t('worker group')" />
+            <el-table-column prop="status" sortable :sort-orders="['ascending', 'descending']" :label="$t('state')" />
+            <el-table-column prop="heartbeatDate" sortable :sort-orders="['ascending', 'descending']" :label="$t('date')">
+                <template #default="scope">
+                    <date-ago class-name="text-muted small" :inverted="true" :date="scope.row.heartbeatDate" />
+                </template>
+            </el-table-column>
```

Comment:
Those two columns will be empty when using the Kafka runner, maybe we should have a way to hide them? Or maybe we should hardcode now/UP.
Or maybe just remove them so we took time to handle it properly (I added this to the followup issue already).

Reply:
Those properties are set up by default so they will never be empty